central/main.go

```diff
@@ -655,15 +655,17 @@ func customRoutes() (customRoutes []routes.CustomRoute) {
 		routes.CustomRoute{
 			Route: scannerDefinitionsRoute,
 			Authorizer: perrpc.FromMap(map[authz.Authorizer][]string{
-				or.ScannerOr(user.With(permissions.View(resources.ScannerDefinitions))): {
+				or.SensorOrAuthorizer(
+					or.ScannerOr(
+						user.With(permissions.View(resources.ScannerDefinitions)))): {
 					routes.RPCNameForHTTP(scannerDefinitionsRoute, http.MethodGet),
 				},
 				user.With(permissions.Modify(resources.ScannerDefinitions)): {
 					routes.RPCNameForHTTP(scannerDefinitionsRoute, http.MethodPost),
 				},
 			}),
 			ServerHandler: scannerDefinitionsHandler.Singleton(),
-			Compression:   false,
+			Compression:   true,
```

Comment:
I have tried looking into utilizing https://pkg.go.dev/archive/zip#File.OpenRaw but I couldn't find a clean solution since we'd need to pass more information to Scanner

Reply:
Compression is actually set by the HTTP client. The idea is that after this change, in Scanner, we will set [`DisableCompression` to `true`](https://pkg.go.dev/net/http#Transport) when retrieving the bundle.

Reply:
ah that's your plan. Sounds good to me

Reply:
Just realized that in Sensor I was missing we need to proxy `Accept-Encoding` for this to work. Let me update the PR.